sklearn/preprocessing/tests/test_label.py

```diff
@@ -208,6 +208,11 @@ def test_label_encoder_errors():
     assert_raise_message(ValueError, msg, le.inverse_transform, [-2])
     assert_raise_message(ValueError, msg, le.inverse_transform, [-2, -3, -4])
 
+    # Fail on inverse_transform("")
+    msg = "bad input shape ()"
+    assert_raises(ValueError, le.inverse_transform, "")
```

Comment:
I think assert_raises is unnecessary?

Reply:
Yeah, just wanted to be more specific. Should I remove it?

Reply:
I believe it would be appropriate to fail on `assert_raises` if the error is other than `ValueError` rather than failing in `assert_raises_message`

Reply:
I think assert_raise_message is enough (see other tests), so please remove it unless you can give a more persuasive reason.

Reply:
Alright, as you say